content/blueprints/gcp.md

```diff
@@ -0,0 +1,1277 @@
+
+    *type:* boolean
+    *default:* False
+  * `name`
+    Optional static ip name. By default it will be static ip id.
+
+    *type:* string
+    *default:*
+
+
+
+
+
+## cloudify.gcp.nodes.GlobalForwardingRule
+**Derived From:** [cloudify.nodes.Root]({{< relref "blueprints/built-in-types.md" >}})
+
+A GCP GlobalForwardingRule.
+
+Can only be used in conjunction with a GlobalAddress to set up HTTP and HTTPS forwarding.
+
+
+
+**Properties:**
+
+
+  * `port_range`
+    Port number used by this forwarding rule. If packets are redirected to HTTP proxy, then possible values are 80 and 8080, in case of HTTPS proxy the only accepted value is 443.
+
+    *type:* string
+    *default:* 80
+  * `additional_settings`
+    Additional setting for ssl certificate
+
+    *default:* {}
+  * `name`
+    Optional global forwarding rule name. By default it will be global forwarding rule id.
+
+    *type:* string
+    *default:*
+  * `target_proxy`
+    URL of a target proxy (http or https) that will receive traffic coming from specified IP address.
+
+    *type:* string
+    *default:*
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `ip_address`
+    IP address associated with this forwarding rule. This address should be reserved earlier.
+
+    *type:* string
+    *default:*
+  * `use_external_resource`
+    Indicate whether the resource exists and use existing (true) or if Cloudify should create new resource (false).
+
+    *type:* boolean
+    *default:* False
+
+
+
+
+
+## cloudify.gcp.nodes.HealthCheck
+**Derived From:** [cloudify.nodes.Root]({{< relref "blueprints/built-in-types.md" >}})
+
+A GCP HealthCheck.
+
+This describes a method that a TargetProxy can use to verify that particualr backend Instances are functioning. Backends which fail the health check verification will be removed from the list of candidates.
+
+
+
+**Properties:**
+
+
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `additional_settings`
+    Optional additional settings. Possible fields in dictionary are: port, request_path, timeout_sec, check_interval_sec, healthy_threshold, unhealthy_threshold.
+
+    *default:* {}
+  * `health_check_type`
+    This field indicates if this health check is a HTTP or HTTPS based health check. Possible values are: 'http' and 'https'.
+
+    *type:* string
+    *default:* http
+  * `use_external_resource`
+    Indicate whether the resource exists and use existing (true) or if Cloudify should create new resource (false).
+
+    *type:* boolean
+    *default:* False
+  * `name`
+    Optional health check name. By default it will be health check id.
+
+    *type:* string
+    *default:*
+
+
+
+
+
+## cloudify.gcp.nodes.Image
+**Derived From:** [cloudify.nodes.Root]({{< relref "blueprints/built-in-types.md" >}})
+
+A stored image which can be used as the base for newly created Instances.
```

Comment:
A stored image ~~which~~ **that** can be used as the base for newly created Instances.